pkg/app/piped/planner/terraform/terraform.go

```diff
@@ -81,6 +81,23 @@ func (p *Planner) Plan(ctx context.Context, in planner.Input) (out planner.Outpu
 		return
 	}
 
+	// convert module source to URL.
+	for _, f := range files {
+		for _, m := range f.Modules {
+			if m.IsLocal {
+				l := provider.NewLocalModuleSourceConverter(in.GitPath.Repo.Remote, in.GitPath.Repo.Branch, ds.RepoDir, ds.AppDir)
+				url, err := l.MakeURL(m.Source)
+				if err != nil {
+					return out, err
+				}
+
+				m.Source = url
+			} else {
+				// TODO: convert remote module to URL.
+			}
+		}
+	}
```

Comment:
This shouldn't be included in this function. I think we should pass a GitPath to FindArtifactVersions to handle inside that function, instead.

Reply:
I see I'll fix it! Thank you!
`ds.RepoDir`, `ds.AppDir` are also needed, so I'll fix to pass a `GitPath` and a `ds`!